tests/SIL.LCModel.Core.Tests/Scripture/BCVRefTests.cs

```diff
@@ -18,20 +19,52 @@ namespace SIL.LCModel.Core.Scripture
 	[TestFixture]
 	public class BCVRefTests
 	{
-		/// ------------------------------------------------------------------------------------
 		/// <summary>
-		/// Initializes the VersificationTable class for tests.
+		/// Initialize some testing files and variables, to prepare VersificationTable for test runs. Should only be done once.
 		/// </summary>
-		/// ------------------------------------------------------------------------------------
+		/// <remarks>
+		/// We assert that if these files exist they are the same size as the expected data. Parallel test execution can cause
```

Comment:
> `size`

Looks like we're verifying that they're the same, not just the same size.

Reply:
Oh yeah, I got ambitious.